### DISCUSSION TITLE
Is it possible to track right click on TrayIcon?

### DISCUSSION THREAD

**Discussion by demchenkov** [AvaloniaUI/Avalonia#15635] [Q&A]
I'm just joined Avalonia community, so probably it may be silly question:

What I want to do: Show/Hide my application on left and right click on tray icon.

Now wrote such code: 
```axaml
<TrayIcon.Icons>
  <TrayIcons>
    <TrayIcon Icon="/Assets/icon.ico"
              ToolTipText="Avalonia Tray Icon ToolTip"
              Command="{Binding ShowAppCommand}">
    </TrayIcon>
  </TrayIcons>
</TrayIcon.Icons>
```

Command and Event work fine, but only for left mouse click. 
Is it possible somehow make it work for right click? Maybe some workarounds? 

I also tried to add something like: 
```axaml
<TrayIcon.Menu>
  <NativeMenu Opening="NativeMenu_OnOpening">
    <NativeMenuItem Header="Settings" />
  </NativeMenu>
</TrayIcon.Menu>
```

But `Opening` event never rises on my Win 11 env. Seems like a bug

I'll be appreciate for your support

**Comment by timunie** [comment]
TrayIcons seems to be limited to what is possible on all platforms. 
related: https://github.com/AvaloniaUI/Avalonia/issues/10269

**Reply by demchenkov** [reply]
Is it possible to add right click handling only for Windows? 
How can I achieve it? 
Samples are welcome
